libraries/ESP8266WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -15,16 +15,55 @@ class FunctionRequestHandler : public RequestHandler {
     , _uri(uri)
     , _method(method)
     {
+		int numParams = 0, start = 0;
+        do {
+            start = _uri.indexOf("{}", start);
+            if (start > 0) {
+                numParams++;
+                start += 2;
+            }
+        } while (start > 0);
+        pathArgs = std::vector<String>(numParams);
```

Comment:
Is it better than `pathArgs.resize(numParams)` ?

Reply:
Changed to ```pathArgs.resize(numParams)```